deps/histogram/README.md

```diff
@@ -1,3 +1,78 @@
-# HdrHistogram_c
+HdrHistogram_c: 'C' port of High Dynamic Range (HDR) Histogram
```

Comment:
This file was previously written by us and not pulled in from upstream. The upstream file has no reference to the upstream repository.